min_k8s_version/README.md

```diff
@@ -0,0 +1,13 @@
+# Minimum Kubernetes Version
```

Comment:
can you also add this to the top-level README?

Reply:
Do you mean the main README here: https://github.com/tilt-dev/tilt-extensions/blob/master/README.md ?

Reply:
yes